### PR TITLE
Feature/front Battery Module page

### PR DESCRIPTION
**Pull request by amalia94**
Created cards and dropdowns with parameters and methods for each service.

### CODE REVIEW

File: src/front_end/carsdata/app/components/animations/batteryState.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      return (
+            <lord-icon
+    src="https://cdn.lordicon.com/idhdbssm.json"
+    trigger="hover"
+    class="my-lordicon">
+</lord-icon>
+      );
+}
+
+export default BatteryStateOfCharge
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/animations/batteryState.js:13]: fix the indentation

File: src/front_end/carsdata/app/components/animations/batteryVoltage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            src="https://cdn.lordicon.com/hhoxkrtv.json"
+            trigger="hover"
+            class="my-lordicon">
+        </lord-icon>
+      );
+}
+
+export default BatteryVoltage
+
+
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/animations/batteryVoltage.js:15]: fix the indentation

File: src/front_end/carsdata/app/components/animations/lifeCycle.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+            <lord-icon
+            src="https://cdn.lordicon.com/abaxrbtq.json"
+            trigger="hover"
+            class="my-lordicon">
+        </lord-icon>
+      );
+}
+
+export default LifeCycle
+
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/animations/lifeCycle.js:14]: fix the indentation

File: src/front_end/carsdata/app/batteryModule/page.tsx [hunk 4]
```diff
@@ -1,27 +1,167 @@
...
+                </div>
+                <div className=" flex items-center justify-center">
+                    <table>
+                        <tr>
+                            <td>
+                                <div className="card w-96 bg-green-300 text-black">
+                                    <div className="card-body">
+                                        <h2 className="card-title">Battery</h2>
+                                        <h1>Battery level: {jsonData.query_params.battery_level}</h1>
+                                        {/* <p>If a dog chews shoes whose shoes does he choose?</p> */}
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/batteryModule/page.tsx:66]: please delete commented cod from this file

**Review by Elisei-26 — CHANGES_REQUESTED** [review]

**Review by Elisei-26 — APPROVED** [review]

**Comment by GeorgeCimpoeru** [comment]
Please solve the conflicts

**Review by GeorgeCimpoeru — APPROVED** [review]

**Pull request merged**